pkg/config/app/kuma-dp/config.go

```diff
@@ -142,6 +142,10 @@ type DataplaneRuntime struct {
 	Resources DataplaneResources `json:"resources,omitempty"`
 	// SocketDir dir to store socket used between Envoy and the dp process
 	SocketDir string `json:"socketDir,omitempty" envconfig:"kuma_dataplane_runtime_socket_dir"`
+	// MetricsCertPath path to the certificate for metrics listener
+	MetricsCertPath string `json:"metricsCertPath,omitempty" envconfig:"kuma_dataplane_metrics_cert_path"`
+	// MetricsKeyPath path to the key for metrics listener
+	MetricsKeyPath string `json:"metricsKeyPath,omitempty" envconfig:"kuma_dataplane_metrics_key_path"`
```

Comment:
Would you consider grouping Metrics in a dedicated Object with the prefixed properties embedded ?